Docs: warn macOS users against installing RabbitMQ via Homebrew (#7267)
aiidateam/aiida-core: docs/source/installation/guide_complete.rst

            The service has to manually be started each time the machine reboots.

        .. warning::

>             The ``brew install`` can sometimes install a version of Erlang that is incompatible with the installed version of RabbitMQ, causing ``rabbitmq-server`` to crash on startup with no clear error message.

Reviewer: ```suggestion
           Homebrew can sometimes install a version of Erlang that is incompatible with the installed version of RabbitMQ.
```
